Mainmenu

rigeljoseph opened the pull request:
hello! I added the main menu with basic functionalities, without UI Design..

rigeljoseph committed:
mainmenu

rigeljoseph committed:
Revert "mainmenu"

This reverts commit 2aa35ea8084eaa2732d425d35b67c3cd59057a6f.

rigeljoseph committed:
Main Menu

Can transfer layouts
Displays Highscore

Needs UI Fixing
Needs to put Settings Functionality

nmvNoble reviewed (changes requested):
Most of them work, the Exit Game doesn't for me. If we could make it so that the Highscore is displayed immediately and to leave space for the Rungeon Logo.

rigeljoseph committed:
Update

changed the high score.

problem with settings. when i go back to main menu it goes back to unchecked

rigeljoseph committed:
fixed highscore

rigeljoseph committed:
menu

rigeljoseph committed:
Revert "menu"

This reverts commit 542b52ffefbb092252467d1e863ad85ad57dc0f2.

rigeljoseph committed:
Revert "menu"

This reverts commit 542b52ffefbb092252467d1e863ad85ad57dc0f2.
